### PR TITLE
Specs regions

### PR DESCRIPTION
**Pull request by andru1989**
Regions features finished for create, update and destroy

### CODE REVIEW

File: spec/support/region_macros.rb [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
+
+    click_button 'Update Region'
+  end
+
+  def destroy_region
+    region_index
+    click_link('Destroy', match: :first)
+  end
+
+  def region_index
```
Comment (orendon) [inline spec/support/region_macros.rb]: try to use explicit and descriptive names like `navigate_to_region_index`

File: spec/features/regions_spec.rb [hunk 2]
```diff
@@ -17,10 +17,27 @@
...
+    2.times do
+      expect( create_region(region) )
+    end
+
     expect(page).to have_content 'Name has already been taken'
+  end
 
+  scenario 'edits a region' do
+    create(:region)
+    update_region
```
Comment (orendon) [inline spec/features/regions_spec.rb]: I don't see any particular reason to extract `update_region` and `destroy_region` into a helper macro.

In general we only need to extract when we start seeing duplication, for now I think is better to have those feature specs explicitly defined on each rspec scenario

File: spec/support/region_macros.rb (hunk 1 above)
Comment (gomayonqui) [inline spec/support/region_macros.rb]: :+1:

**Comment by orendon** [comment]
good job :+1:

**Pull request merged**